modes/play/objects/groundTile.js

```diff
@@ -1,5 +1,6 @@
 class GroundTile {
   constructor(x, y) {
+    this.name = "GroundTile";
```

Comment:
Started adding names to classes. I _think_ this is the easiest way for one object to check "What kind of thing am I running into right now?" For example, collision callbacks just receive some obj. If the callback behaves differently based on whether it's a Player or a GroundTile, it can just ask its name to find out which it is.